cves/CVE-2014-3583.yml

```diff
@@ -190,8 +213,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: Vulnerability was caused by trusting that external input already contained a null char '\0'.
```

Comment:
This would also be good to include in the description (about the null char) to add some more clarity.